File: app/src/main/java/com/georgcantor/wallpaperapp/ui/util/TagViewHolder.java
package com.georgcantor.wallpaperapp.ui.util;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.georgcantor.wallpaperapp.R;

public class TagViewHolder extends RecyclerView.ViewHolder {

    public TextView tag;

    public TagViewHolder(View itemView) {
        super(itemView);
        tag = (TextView) itemView.findViewById(R.id.tag_item);
    }
}

File: app/src/main/java/com/georgcantor/wallpaperapp/ui/util/TagViewHolder.kt
package com.georgcantor.wallpaperapp.ui.util

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.tag_item.view.*

class TagViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var tag: TextView = itemView.tag_item
}
